examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -27,17 +27,21 @@
           // event.
           context.sendResponseToCreative({'collected-data': 'abc'});
         });
-      };
+      }
```

Comment:
think you still need a ; here, no?

Reply:
Yeah, you're right. Will fix. (Wonder why the linter didn't mind?)